doc/doxygen/riot.doxyfile

```diff
@@ -826,9 +828,10 @@ EXCLUDE_PATTERNS       = */board/*/tools/* \
                          */drivers/nrf24l01p/include/nrf24l01p_settings.h \
                          */pkg/*/*/* \
                          */pkg/tlsf/patch.txt \
+                         */sys/include/embUnit/* \
                          */sys/random/tinymt32/* \
                          */toolchain/* \
-                         */vendor/* \
+                         */vendor/*
```

Comment:
Leave the backslash in, in my experience it reduces the risk of merge collisions when modifying the list in later PRs.